spikeextractors/extractors/mearecextractors/mearecextractors.py

```diff
@@ -73,12 +78,12 @@ def get_sampling_frequency(self):
     def get_traces(self, channel_ids=None, start_frame=None, end_frame=None):
         if np.any(np.diff(channel_ids) < 0):
             sorted_idx = np.argsort(channel_ids)
-            recordings = self._recordings[np.sort(channel_ids), start_frame:end_frame]
+            recordings = self._recordings[start_frame:end_frame, np.sort(channel_ids)]
             return recordings[sorted_idx]
         else:
             if sorted(channel_ids) == channel_ids and np.all(np.diff(channel_ids) == 1):
                 channel_ids = slice(channel_ids[0], channel_ids[0] + len(channel_ids))
-            return self._recordings[channel_ids, start_frame:end_frame]
+            return self._recordings[start_frame:end_frame, channel_ids]
```

Comment:
@samuelgarcia you need to transpose this

(until we change internally get_traces shape)